package.json

```diff
@@ -23,14 +26,8 @@
     "eslint-plugin-react": "7.16.0",
     "eslint-plugin-react-hooks": "2.2.0"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/pmedianetwork/eslint-config.git"
+  "devDependencies": {
+    "mocha": "7.1.0"
   },
-  "author": "",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/pmedianetwork/eslint-config/issues"
-  },
-  "homepage": "https://github.com/pmedianetwork/eslint-config#readme"
+  "license": "ISC"
 }
```

Comment:
can we add prettier and linting to this repo now that we are actually writing code here? Not sure if we could lint it with "ourselves" .... linting the eslint repo _with_ the eslint repo is kinda meta 😅

Reply:
We can - I was thinking about it, but I wasn't sure if we event are fine with this massive refactor 😄 
I'd probably do it later in future as a separate issue, but that would be once we write more code here. What do you think?

Reply:
okay, lets create a follow-up

Reply:
No need
https://github.com/pmedianetwork/eslint-config/pull/64/commits/9eca8320ec15989166ad516f413a5bc36980c0ca

Reply:
wow, you are so fast! But I think we need the prettier-config that we have everywhere: single quotes as strings, no semicolons etc. We have this in `insights` in `package.json`:

```
  "prettier": {
    "printWidth": 120,
    "semi": false,
    "singleQuote": true,
    "trailingComma": "all",
    "tabWidth": 4
  },
```

Reply:
👍 
https://github.com/pmedianetwork/eslint-config/pull/64/commits/a7455fda1e10bec289f02eb96f385204cd4813d7